tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -129,25 +107,58 @@ private async Task InitializeAsync()
                 Log.Error(e, "Initializing Live Debugger failed.");
             }
 
+            bool ShouldInitialize()
+            {
+                lock (GlobalLock)
+                {
+                    if (_isInitialized)
+                    {
+                        Log.Information("Live Debugger is already initialized.");
+                        return false;
+                    }
+
+                    if (!_settings.Enabled)
+                    {
+                        Log.Information("Live Debugger is disabled. To enable it, please set DD_DEBUGGER_ENABLED environment variable to 'true'.");
+                        return false;
+                    }
+
+                    if (_settings.TransportType != TracesTransportType.Default)
+                    {
+                        Log.Information("Live Debugger is not supported on UDS or named pipelines.");
+                        return false;
+                    }
+
+                    var isConfigurationSupported = !string.IsNullOrWhiteSpace(_discoveryService.ConfigurationEndpoint);
```

Comment:
Wrong check here. This check should display a warning that says "You must enable Remote Configuration Management in datadog-agent in order to leverage Live Debugger. All debugging features will be disabled.

We should add another check, for `datadog-agent version as returned by the /info response < 7.38` , for the existing warning message. Note that the check for datadog-agent version must come BEFORE the check for RCM enablement.

Reply:
I suggest to change log, so it will be more explicit, see dd4b6f65cef327ba5580fa8e82d51d809eaa4a4d